buildtest/cli/compilers.py

```diff
@@ -151,14 +153,17 @@ def compiler_find(configuration, modulepath=None, detailed=None, update=None):
     # print out all compilers from existing configuration file
     # run buildtest config compilers find --update to update existing configuration file
 
+    if filepath and filepath.split(".")[-1] != ".yml":
+        raise ValueError("alternative file specified should be a yml file")
+
```

Comment:
i think we should also check for edge case and raise exception when `--file` specifies a directory path `i.e buildtest cg co find -u /tmp` should raise an exception.

As per the configuration file we are not forcing `.yml` extension right now so i am not sure if we should force this check.